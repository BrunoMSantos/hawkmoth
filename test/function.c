/**
 * Foo function.
 */
int foo(int bar, int baz);
