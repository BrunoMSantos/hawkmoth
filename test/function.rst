
.. c:function:: int foo(int bar, int baz)

   Foo function.


.. c:function:: int fooing()

   Void parameter.


.. c:function:: int fooation()

   Empty parameters.

