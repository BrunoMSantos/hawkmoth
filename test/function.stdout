.. c:function:: int foo(int bar, int baz)

   Foo function.
   

